### DISCUSSION TITLE
Add Ability for Sync to be a Command Too

### DISCUSSION THREAD

**Discussion by michal-sadurski** [mahyarmirrashed/yaos#7] [Ideas]
Hi!
I would love to have Sync available as a command just like "Show unmerged/conflicting files".

**Comment by mahyarmirrashed** [comment]
Hello Michal and thank you for creating this discussion thread. Could you please provide me with a bit more context into what you're looking for? A bit of a more thorough description, drawings, or sketches of what you're looking for can help me and others who may view this thread to better understand what you're envisioning.

Looking forward to hearing your response!

**Reply by michal-sadurski** [reply]
Hi, thanks for reply :)

When i open command palette (Cmd+P) i'd like to have sync command there, like that:
![image](https://github.com/mahyarmirrashed/yaos/assets/9074408/fa443c19-8453-43bf-b68e-2ed43b7fb770)

And i believe it would be then available for hotkey assignment
![image](https://github.com/mahyarmirrashed/yaos/assets/9074408/e89affe9-1296-464d-9fcc-70867338941b)

**Reply by mahyarmirrashed** [reply]
Ok, gotcha, this makes sense. Sure, I think I should be able to make that change pretty quickly.

**Reply by mahyarmirrashed** [reply]
Hi @michal-sadurski , to update you, this should be completed in latest release (v0.7.0) of the plugin.

**Reply by michal-sadurski** [reply]
Thanks, that was quick indeed :) And it works like a charm.
